code/1_cleaning/09_merge_grace_virtwflows.R

```diff
@@ -0,0 +1,71 @@
+# Description: This file merges GRACE cells with agricultural employment share
```

Comment:
@cheikhfall07 I think this is an incorrect header, maybe accidentally copied over from cleaning script 08?

Reply:
Agreed! Thanks for spotting this one!